- floor: 8
  altitude: 30.8
  map_name: '/eng2/8f'
- floor: 7
  altitude: 26.9
  map_name: '/eng2/7f'
- floor: 6
  altitude: 22.9
  map_name: '/eng2/6f'
- floor: 5
  altitude: 17.5
  map_name: '/eng2/5f'
- floor: 4
  altitude: 14.1
  map_name: '/eng2/4f'
- floor: 3
  altitude: 9.3
  map_name: '/eng2/3f'
- floor: 2
  altitude: 4.4
  map_name: '/eng2/2f'
- floor: 1
  altitude: 0
  map_name: '/eng2/1f'
